trace_replay/trace_replay.cc

```diff
@@ -529,14 +528,19 @@ Status Replayer::Replay() {
     if (trace.type == kTraceWrite) {
       if (trace_file_version_ < 2) {
         WriteBatch batch(trace.payload);
-        db_->Write(woptions, &batch);
+        s = db_->Write(woptions, &batch);
+        if (!s.ok()) {
+          break;
+        }
```

Comment:
Are all these break statements really necessary?  Isn't this all inside a "while s.ok" loop?

Reply:
As the body of the loop is quite large - I do think it actually makes the intention much clearer